### PR TITLE
Eliminate Plots as dependency in Docs; use Makie

### PR DESCRIPTION
**Pull request by navidcy**
Follow up to #2553 and #2556.

Deals with partly of what's discussed in #2326 and #2448, namely use Makie for plots.

### CODE REVIEW

File: examples/langmuir_turbulence.jl [hunk 1]
```diff
@@ -263,14 +263,17 @@ xw, yw, zw = nodes(time_series.w)
 xu, yu, zu = nodes(time_series.u)
 nothing # hide
 
-# Finally, we're ready to animate using Makie.
+# We are now ready to animate using Makie. # We use Makie's `Observable` to animate
```
Comment (francispoulin) [inline examples/langmuir_turbulence.jl:266]: Do you want two #'s on this line?

File: examples/ocean_wind_mixing_and_convection.jl [hunk 2]
```diff
@@ -239,60 +241,85 @@ time_series = (w = FieldTimeSeries(filepath, "w"),
...
 nothing # hide
 
-# We start the animation at `t = 10minutes` since things are pretty boring till then:
+# We start the animation at ``t = 10minutes`` since things are pretty boring till then:
 
 times = time_series.w.times
 intro = searchsortedfirst(times, 10minutes)
 
-anim = @animate for (i, t) in enumerate(times[intro:end])
+# We are now ready to animate using Makie. # We use Makie's `Observable` to animate
```
Comment (francispoulin) [inline examples/ocean_wind_mixing_and_convection.jl:268]: Same comment as before

**Review by francispoulin — APPROVED** [review]
Nicely done!

**Comment by navidcy** [comment]
Can't seem to be able to put the instantaneous buoyancy flux circle on the top-right plot for the `convecting_plancton` example... Any ideas?

**Comment by francispoulin** [comment]
> Can't seem to be able to put the instantaneous buoyancy flux circle on the top-right plot for the `convecting_plancton` example... Any ideas?
> 
> This
> 
> https://github.com/CliMA/Oceananigans.jl/blob/51b27bd55abc8931fdae618ed82881e777c7b6f2/examples/convecting_plankton.jl#L288-L289
> 
> works but when I record the circle is not there!

Do you think this is a bug with `Makie`?

**Comment by navidcy** [comment]
No bug @francispoulin. I just had a mistake in my logic.  9191c0d fixed it.

**Comment by navidcy** [comment]
no bug @francispoulin. I just had a mistake in my logic. 
9191c0d fixes it. Thanks!

**Comment by navidcy** [comment]
I just read this blog post and got excited:

https://github.blog/2022-05-19-math-support-in-markdown/

let's see...

$$
\begin{equation}
\partial_t \mathbf{u} + \ mathbf{u} \cdot \mathbf{\nabla} \mathbf{u} = - \ mathbf{\nabla}p + \nu \nabla^2 u
\end{equation}
$$

**Comment by glwagner** [comment]
NICE!

**Comment by francispoulin** [comment]
This definitely improves my quality of life.  Wonderful!

**Pull request merged**